android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -251,9 +266,43 @@ public void cleanUpResources() {
         stopPlayback();
     }
 
+    private void releaseMuxPlayer() {
```

Comment:
not calling this function abhi crashing on muxStatsExoPlayer.release().
Let me know when u r free, we can sit on this